app/assets/javascripts/pretty-text/addon/oneboxer.js

```diff
@@ -1,3 +1,4 @@
+import domFromString from "discourse-common/lib/dom-from-string";
```

Comment:
You'll have to add `pretty-text` to `discourse-common` dependencies

Reply:
why that ?

Reply:
guess I will just add this to core discourse